main/src/cgeo/geocaching/connector/gc/GCWebAPI.java

```diff
@@ -240,6 +248,12 @@ Geopoint toCoords() {
     @JsonIgnoreProperties(ignoreUnknown = true)
     static final class CallerSpecific {
         boolean favorited;
+
+        CallerSpecific(final boolean favorited) {
+            this.favorited = favorited;
+        }
+
+        CallerSpecific() {}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['}' is not preceded with whitespace.](https://app.codacy.com/manual/cgeo/cgeo/pullRequest?prid=5262679)